db/migrate/20200720085855_rename_column_on_users_from_role_to_designation.rb

```diff
@@ -0,0 +1,5 @@
+class RenameColumnOnUsersFromRoleToDesignation < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :users, :role, :designation
```

Comment:
Maybe a better rename here would be to `title` or `description` ?

Reply:
Can we ask Tanushree what users use this field for? Is it useful?